src/content/docs/accounts/accounts/automated-user-management/onelogin-scimsso-application-configuration.mdx

```diff
@@ -92,28 +95,50 @@ In the left pane, select **Provisioning** and complete the following:
 3. Set **When users are deleted in OneLogin, or the user's app access is removed, perform the below action** to **Delete**.
 4. Set **When user accounts are suspended in OneLogin, perform the following action** to **Suspend**.
 
-### Step 5. Save your changes [#save]
+### Fill in the Parameters form [#parameters-form]
+
+In the left pane, select **Parameters** and complete the following:
+
+1. Click **Groups** field.
+
+   ![Screenshot of the OneLogin Parameter form showing which fields to configure.](./images/onelogin_parameter_form_general.png "onelogin_parameter_form_general.png")
+2. Check **Include in User Provisioning**.
+
+   ![Screenshot showing the OneLogin Parameter details page.](./images/onelogin_parameter_form_specific.png "onelogin_parameter_form_specific.png")
+3. Click **Save**.
+
+### Save your changes [#save]
 
 After you complete the above forms, click **Save**. Then, return to the **Configuration** form and enable the API connection.
 
-## Assign users [#configure-sso]
+### Step 4. Assign users [#configure-sso]
 
 After New Relic SCIM/SSO application configuration is finished and New Relic side configuration is finished, you can begin to assign users to the application.
 
-Assign the New Relic SCIM/SSO application to a user.
+Assign the New Relic SCIM/SSO application to a user: 
 
 1. Go to the OneLogin web site and sign in with an account that has administrator permissions.
 2. From the OneLogin home page, click **Administration**.
 3. From the OneLogin Administration page, choose the **Users** menu Users item.
 4. From the OneLogin Users page, click the user you want to assign the application to.
 5. From the user's page, click **Applications**.
 6. From the user's application page, click the plus sign and select the "New Relic by Organization" application.
-7. From the **Edit New Relic by Organization login for user** page, enter the user's time zone in IANA Time Zone database format (also known as the Olson time zone database format) and click **Save**.
+7. Important: Updating users' time zones is important, as charts and other user assets display times. Default is UMT. From the **Edit New Relic by Organization login for user** page, enter the user's time zone in IANA Time Zone database format (also known as the Olson time zone database format) and click **Save**.
 8. If you're using **Roles** to define your New Relic capability groups, from the user's application page, click the proper role(s) for the user and then click **Save User**.
 
-OneLogin provisions users in near real time so almost the moment you save the user in OneLogin, the user should be ready to use at New Relic.
+### Step 5. Downgrade some users to basic [#downgrade]
```

Comment:
Should this be an H2 ## rather than an H3 ###?

Reply:
it was a few nested h3s under an h2